stripper/ze_einstein_v7.cfg

```diff
@@ -11,4 +11,19 @@ add:
 	"nodmgforce" "0"
 	"spawnflags" "8"
 	"StartDisabled" "0"
+}
+
+;Filter item buttons to their holders
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "modified_george_cade_sb_buttonRunScriptCodeInputUse<-function(){if(self.GetMoveParent().GetMoveParent()==activator){return true;}else{return false;}}51"
+		"OnMapSpawn" "sexy1_buttonRunScriptCodeInputUse<-function(){if(self.GetMoveParent().GetMoveParent()==activator){return true;}else{return false;}}51"
+		"OnMapSpawn" "FlamethrowerButtonRunScriptCodeInputUse<-function(){if(self.GetMoveParent().GetMoveParent()==activator){return true;}else{return false;}}51"
```

Comment:
Target these buttons with `targetname*`, they're spawned from templates and so will each have unique names. E.g. `modified_george_cade_sb_button&0001`, `modified_george_cade_sb_button&0002`.

Reply:
Actually tested this change beforehand and it worked, so I assume this just isn't needed with the entwatch max amounts being correct, but will leave in just in case the filtering for the DZ entwatch is different from GFL's (though I doubt it).